MarsRover.Tests/InputParserShould.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using FluentAssertions;
+using Xunit;
+
+namespace MarsRover.Tests
+{
+    public class InputParserShould
+    {
+        [Theory]
+        [InlineData("N", Direction.North)]
+        [InlineData("n", Direction.North)]
+        [InlineData("S", Direction.South)]
+        [InlineData("s", Direction.South)]
+        [InlineData("E", Direction.East)]
+        [InlineData("e", Direction.East)]
+        [InlineData("W", Direction.West)]
+        [InlineData("w", Direction.West)]
+        public void ReturnRoverInitialDirection_GivenValidString(string input, Direction expected)

```

Comment:
Great test. I think the name implies coupling with the Rover in some way, but it really just provides a Direction given a valid letter equivalent. The parser doesn't care what you do with the Direction (in this case, instantiation of a Rover)